scripts/deploy_subscription_manager.py

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/python3
-from brownie import Contract, SubscriptionManager, Wei, project
-
+from ape import project
 from scripts.utils import get_account
+from web3 import Web3
 
-INITIAL_FEE_RATE = Wei("1 gwei")
+INITIAL_FEE_RATE = Web3.to_wei(1, "gwei")
```

Comment:
Can use `ape.convert(“1 gwei", int)`, or a specific token if you have the `ape_tokens` plugin installed

Reply:
I like how it looks, especially with tokens.
I didn't find how to convert data to bytes, int to bytes for example. Could you point where to look at?
And another question: can be used something like that `convert("1_000_000_000 T", int)` in tests before token deployment?

Reply:
We don't have support for bytes like this right now, it's best to use `eth_utils` for this type of conversion. Our converter is primarily for customized conversions from strings to native python types, prior to abi conversion (per ecosystem)

Reply:
Also yes, you should be able to use underscores since we pass it to `Decimal` which should accept that

Reply:
Underscores work, I meant that token not in the list of tokens yet (in test before deploying). In that case how to use `ape-tokens` to run it? (rn I get `ape.exceptions.ConversionError: Must be connected to a provider to use the token converter`) with line `convert("1_000_000_000 NU", int)`